src/snowflake/snowpark/session.py

```diff
@@ -452,6 +464,16 @@ def sql_simplifier_enabled(self, value: bool) -> None:
             pass
         self._sql_simplifier_enabled = value
 
+    @custom_packages_upload_enabled.setter
+    @experimental_parameter(version="1.6.0")
+    def custom_packages_upload_enabled(self, value: bool) -> None:
```

Comment:
How about adding docstring here too to explain what this is for?

Reply:
Sure, added

Reply:
Docstrings are moved to the function which ends up being rendered (the setter)